packages/go/analysis/ad/esc_shared.go

```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/specterops/bloodhound/dawgs/util/channels"
 	"github.com/specterops/bloodhound/graphschema/ad"
 	"github.com/specterops/bloodhound/log"
-	"github.com/specterops/bloodhound/slices"
+	slicesext "github.com/specterops/bloodhound/slicesext"
```

Comment:
Another possible redundant alias, likely from rename shenanigans

Reply:
That's exactly what it was. Was doing a multi-step refactor and it just didn't get updated properly after